main/main.cpp

```diff
@@ -997,15 +997,15 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	// Initialize user data dir.
 	OS::get_singleton()->ensure_user_data_dir();
 
-	GLOBAL_DEF("memory/limits/multithreaded_server/rid_pool_prealloc", 60);
+	GLOBAL_DEF_RST("memory/limits/multithreaded_server/rid_pool_prealloc", 60);
```

Comment:
If there are more settings that need the change, maybe this one could be put into another PR addressing a bunch of misc settings, given that this one doesn't belong to the category the PR title suggests.